apps/chat/src/backend/serverless/template.yaml

```diff
@@ -300,7 +298,7 @@ Resources:
     Properties:
       Handler: "index.handler"
       Role: !GetAtt LambdaExecuteRole.Arn
-      Runtime: "nodejs12.x"
+      Runtime: nodejs14.x

```

Comment:
Could we do node 18 now? 14 is on the way out already :)

Reply:
Lambda does not support 18 but 16 is supported I think.